ci/docker/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Prep CI Enviroment
+if [[ "${CI}" = "true" ]]; then
```

Comment:
This is so it is easy to read help on local build or a build from local PC. Not as verbose on output with the set command. CI is standard environment variable for most CI platforms. This tells that it is running on CI platform like Travis. It could be Gitlab, or some other CI platform. Travis says it is on Travis.